gpii/node_modules/accessRequester/test/AccessRequesterTests.js

```diff
@@ -0,0 +1,226 @@
+/*!
+GPII Access Requester
+
+Copyright 2017 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii"),
+    kettle = require("kettle"),
+    nock = require("nock");
+
+require("accessRequester");
+// To reuse gpii.tests.oauth2.invokePromiseProducer()
+fluid.require("%universal/gpii/node_modules/gpii-oauth2/gpii-oauth2-datastore/test/js/DataStoreTestsUtils.js");
+
+kettle.loadTestingSupport();
+
+fluid.registerNamespace("gpii.tests.accessRequester");
+
+gpii.tests.accessRequester.hostname = "http://gpii.net";
+gpii.tests.accessRequester.url = "/access_token";
+gpii.tests.accessRequester.gpiiToken = "gpii-token-test";
+
+gpii.tests.accessRequester.testCases = {
```

Comment:
I think I would split this test data up and relocate to be closer to each test sequence below. Each of the 3 test cases have different sequences so to understand a test, one must look both at its sequence and at the test data in this object.

And by the time you get to the 3rd case, that's quite a bit of scrolling.

You could split the test data up and either move into each testCaseHolder or testEnvironment, or still keep in a separate object but split up and located closer to each relevant test.